### PR TITLE
[hold on]add directive in containerregistry ts.md

### PR DESCRIPTION
**Pull request by kazrael2119**
don't merge this pr until https://github.com/Azure/azure-rest-api-specs/pull/24426 merged to main

### CODE REVIEW

**Review by qiaozha — APPROVED** [review]

File: specification/containerregistry/resource-manager/readme.typescript.md [hunk 1]
```diff
@@ -17,4 +17,30 @@ typescript:
   package-name: "@azure/arm-containerregistry"
   output-folder: "$(typescript-sdks-folder)/sdk/containerregistry/arm-containerregistry"
   generate-metadata: true
+
+directive:
```
Comment (visingla-ms) [inline specification/containerregistry/resource-manager/readme.typescript.md:21]: Not sure why does this need ARM Review.

**Comment by kazrael2119** [comment]
don't merge this pr until we have js release request

**Pull request merged**